### PR TITLE
Fixed issues: issue 2, issue 3, issue 4. Also fixed 'contributing' file, 'README' file, minor bugs and was made small refactoring.

### PR DESCRIPTION
**Pull request by igor90007**
Fixed issues: issue 2, issue 3, issue 4. Also fixed contributing file, readme file, minor bugs and was made small refactoring.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via an issue or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+## Pull Request Process
+1. Code review results exist either as draft pull requests or as issues.
+2. Merge requests possible only after code review.
+3. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
+   do not have permission to do that, you may request the second reviewer to merge it for you.
+## Code of Conduct
```
Comment (ddzirt) [inline contributing.md]: And where is a Code of Conduct? Remove or add.

File: package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
     "react-native-gesture-handler": "^1.1.0",
     "react-native-responsive-screen": "^1.2.2",
     "react-navigation": "^3.9.1",
+    "translate": "^1.1.0",
```
Comment (ddzirt) [inline package.json]: Adding a library just for one method is a very very questionable thing. Explain to me why do we need it exactly?

File: src/components/Text/DefaultText.tsx [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-import { Text } from 'react-native'
+import { StyleProp, Text, TextProps, TextStyle } from 'react-native'
 
 import styles from './styles'
 
-interface Params {
-  text: string
-  additionalStyles: object
+export interface ITextProps extends TextProps {
+  children?: string
```
Comment (ddzirt) [inline src/components/Text/DefaultText.tsx]: TextProps already has children prop, why do override it?

File: src/screens/General/Main.tsx [hunk 4]
```diff
@@ -1,37 +1,31 @@
 import React from 'react'
 import EStyleSheet from 'react-native-extended-stylesheet'
 
-import { Text, View } from 'react-native'
+import { View } from 'react-native'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 
+import { DefaultText } from '../../components/Text'
```
Comment (ddzirt) [inline src/screens/General/Main.tsx]: Please learn how to configure TS to use imports. The goal is to have import like this:

```javascript
import { nameOfImport } from 'directory'
import { nameOfImport } from 'directory/fileName'
```

File: src/components/Text/DefaultText.tsx [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 
-import { Text } from 'react-native'
+import { StyleProp, Text, TextProps, TextStyle } from 'react-native'
 
 import styles from './styles'
 
-interface Params {
-  text: string
-  additionalStyles: object
+export interface ITextProps extends TextProps {
```
Comment (ddzirt) [inline src/components/Text/DefaultText.tsx]: This is not added into index.ts

File: src/components/Text/DefaultText.tsx (hunk 3 above)
Comment (igor90007) [reply src/components/Text/DefaultText.tsx]: agree

File: contributing.md (hunk 1 above)
Comment (igor90007) [reply contributing.md]: ok

**Comment by igor90007** [comment]
> Also, do we really need to separate styles from components if they(styles) are a small file?

Yep, but how to know where we should use one rule and where we have to implement another one. Today there could be simple styles but tomorrow we could need a lot of common styles for Text components.

File: src/screens/General/Main.tsx (hunk 4 above)
Comment (igor90007) [reply src/screens/General/Main.tsx]: @MargaritaZagalets do you have any ideas? Cause I only know "package.json" way (like {"name":"src"} )

File: package.json (hunk 2 above)
Comment (igor90007) [reply package.json]: fixed

File: src/components/Text/DefaultText.tsx (hunk 5 above)
Comment (ddzirt) [reply src/components/Text/DefaultText.tsx]: Add ITextProps to export inside index.ts

File: typings.d.ts [hunk 6]
```diff
@@ -0,0 +1,4 @@
+declare module '*.json' {
+  const value: any
+  export default value
+}
```
Comment (ddzirt) [inline typings.d.ts]: WTF is this ?

**Pull request closed without merging**